packages/migrate/commonsChunkPlugin/__snapshots__/commonsChunkPlugin.test.js.snap

```diff
@@ -0,0 +1,316 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-0" data 1`] = `
+"module.exports = {
+    entry: {
+		app: './src/app.js',
+		vendor: './src/vendors.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                app: {
+                    name: 'app',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
+                    test: 'app'
+                },
+
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
```

Comment:
`minChunks: 2` is incorrect here.

When using `test: "string"`, `minChunks` should not be set.